docs/running-nio/README.md

```diff
@@ -1,10 +1,11 @@
-# Running nio
+# Run nio
 
-nio is a software product that should run throughout your IoT system. These docs will help you figure out how to run nio in whatever way you choose.
+nio facilitates the creation of distributed computing solutions across an almost unlimited array of platforms and devices. As such, there are lots of ways for you to run nio. We've outlined some of the most common implementations below.
```

Comment:
Per the new branding rules, this should be "The nio Platform"

Reply:
done